identifiers/country-us/state-ar-local_gov.csv

```diff
@@ -335,11 +568,18 @@ ocd-division/country:us/state:ar/county:poinsett/council_district:6,Poinsett Cou
 ocd-division/country:us/state:ar/county:poinsett/council_district:7,Poinsett County District 7
 ocd-division/country:us/state:ar/county:poinsett/council_district:8,Poinsett County District 8
 ocd-division/country:us/state:ar/county:poinsett/council_district:9,Poinsett County District 9
+ocd-division/country:us/state:ar/county:poinsett/council_district:10,Poinsett County District 10
+ocd-division/country:us/state:ar/county:poinsett/council_district:11,Poinsett County District 11
+ocd-division/country:us/state:ar/county:polk/council_district:1,Polk County District 1
+ocd-division/country:us/state:ar/county:polk/council_district:2,Polk County District 2
+ocd-division/country:us/state:ar/county:polk/council_district:3,Polk County District 3
+ocd-division/country:us/state:ar/county:polk/council_district:4,Polk County District 4
+ocd-division/country:us/state:ar/county:polk/council_district:5,Polk County District 5
+ocd-division/country:us/state:ar/county:polk/council_district:6,Polk County District 6
+ocd-division/country:us/state:ar/county:polk/council_district:7,Polk County District 7
+ocd-division/country:us/state:ar/county:polk/council_district:8,Polk County District 8
+ocd-division/country:us/state:ar/county:polk/council_district:9,Polk County District 9
```

Comment:
It looks like there should be 11 districts for Polk County: https://www.arcounties.org/counties/polk/